src/routes/webhooks/paddle.ts

```diff
@@ -103,54 +104,75 @@ export const updateUserSubscription = async ({
     return false;
   }
 
-  const user = await con.getRepository(User).findOneBy({ id: userId });
-  if (!user) {
-    logger.error(
-      { provider: SubscriptionProvider.Paddle, data: event },
-      'User not found',
-    );
-    return false;
-  }
-
-  if (user.subscriptionFlags?.provider === SubscriptionProvider.AppleStoreKit) {
-    logger.error(
-      {
-        user,
-        data: event,
+  if (userId === 'anonymous') {
+    const [customer, subscription] = await Promise.all([
+      paddleInstance.customers.get(data.customerId),
+      paddleInstance.subscriptions.get(data.id),
+    ]);
+    await con.getRepository(ClaimableItem).insert({
```

Comment:
Might have to handle if claimable item already exists for email https://github.com/dailydotdev/daily-api/pull/2804/files#r2086337335

Reply:
Reminder to handle this case in regards to https://github.com/dailydotdev/daily-api/pull/2804#discussion_r2088527809